Gemfile

```diff
@@ -8,9 +8,11 @@ gem 'coffee-rails', '~> 4.0.0'
 gem 'jquery-rails'
 gem 'jbuilder', '~> 2.0'
 gem 'bcrypt', '~> 3.1.7'
+gem 'time_ago_in_words', '~> 0.1.1'
 
 group :development, :test do
   gem "rspec-rails", "3.0.1"
   gem 'rspec-collection_matchers', '~> 1.0.0'
   gem "capybara", "2.3.0"
+  gem 'time_ago_in_words', '~> 0.1.1'
```

Comment:
We shouldn't have to install this as a general dependency as well as a test dependency. These two groups are for code that should __only__ be installed on development environments. Try removing this line and double checking that your feature still works.